nagios/tests/test_nagios.py

```diff
@@ -151,6 +144,71 @@ def test_create_event(self):
             'payload:DISK CRITICAL - free space: / 1499 MB (2.46% inode=77%):'
         ]
 
+    def test_event_message_title(self):
+        """
+        Check that right field is used as message title
+        """
+
+        events = [
+            {
+                'type': 'CURRENT HOST STATE',
+                'parts': ['domU-12-31-38-00-78-98', 'UP', 'HARD', '1', 'PING OK - Packet loss = 0%, RTA = 1.03 ms'],
+                'msg_title': 'CURRENT HOST STATE'
+            },
+            {
+                'type': 'CURRENT SERVICE STATE',
+                'parts': ['domU-12-31-38-00-78-98', 'Current Load', 'OK', 'HARD', '1', 'OK - load average: 0.04, 0.03'],
+                'msg_title': 'Current Load'
+            },
+            {
+                'type': 'SERVICE ALERT',
+                'parts': ['domU-12-31-39-02-ED-B2', 'cassandra JVM Heap', 'WARNING', 'SOFT', '1', ''],
+                'msg_title': 'cassandra JVM Heap'
+            },
+            {
```

Comment:
`PASSIVE SERVICE CHECK` event sample is missing

Reply:
Event sample added.